pos-print/src/test/java/com/clouway/pos/print/core/BackgroundReceiptPrintWorkerTest.kt

```diff
@@ -0,0 +1,168 @@
+package com.clouway.pos.print.core
+
+import com.clouway.pos.print.persistent.DatastoreCleaner
+import com.clouway.pos.print.persistent.DatastoreRule
+import com.clouway.pos.print.printer.Status
+import org.jmock.AbstractExpectations.returnValue
+import org.jmock.AbstractExpectations.throwException
+import org.jmock.Expectations
+import org.jmock.Mockery
+import org.jmock.integration.junit4.JUnitRuleMockery
+import org.junit.ClassRule
+import org.junit.Rule
+import org.junit.Test
+import java.io.IOException
+
+/**
+ * @author Tsvetozar Bonev (tsvetozar.bonev@clouway.com)
+ */
+class BackgroundReceiptPrintWorkerTest {
+
+  @Rule
+  @JvmField
+  var context = JUnitRuleMockery()
+
+  companion object {
+    @ClassRule
+    @JvmField
+    val dataStoreRule = DatastoreRule()
+  }
+
+  @Rule
+  @JvmField
+  var cleaner = DatastoreCleaner(dataStoreRule.db())
+
+
+  private fun Mockery.expecting(block: Expectations.() -> Unit) {
```

Comment:
Private function should be last in the file.

Reply:
Moved private functions to the bottom of the file.